src/main/java/co/org/osso/api/CalculatorController.java

```diff
@@ -0,0 +1,41 @@
+package co.org.osso.api;
+
+import com.bugsnag.Bugsnag;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.math.BigInteger;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class CalculatorController {
+
+    private final Calculator calculator;
+    private final Bugsnag bugsnag;
+
+    public CalculatorController(Calculator calculator, Bugsnag bugsnag) {
+        this.calculator = calculator;
+        this.bugsnag = bugsnag;
+    }
+
+    @GetMapping("/fibonacci/{number}")
+    public List<BigInteger> getFibonacci(@PathVariable("number") Integer number){
+        return calculator.getFibonacci(number);
+    }
+
+    @GetMapping("/factorial/{number}")
+    public BigInteger getFactorial(@PathVariable("number") int number){
```

Comment:
Let's return JSON objects always (I.e. use `Map<String, Object>` or, better yet, a POJO)

Reply:
Solving in a new branch